src/contextes/TodosContext.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { createContext, useMemo, useState } from 'react';
+import { TodoContext } from '../types/TodoContext';
+import { Todo } from '../types/Todo';
+import { useLocalStorage } from '../hooks/useLocalStorage';
+
+export const TodosContext = createContext<TodoContext>({
+  todos: [],
+  setTodos: () => { },
+  title: '',
+  setTitle: () => { },
+  filterField: 'All',
+  setFilterField: () => { },
+});
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
use interface prefer